python/taichi/make_changelog.py

```diff
@@ -0,0 +1,98 @@
+# Usage: python3 misc/make_changelog.py [v0.x.y]
+
+import sys
+from git import Repo
```

Comment:
If I understand it correctly, this requires `pip3 install gitpython` as the dependency? If so, should we add it to the deps file?

Reply:
Oh NVM, this was copied over, if it worked, it should still work.

Reply:
make_changelog.py will only be used in development mode, no release mode. See `TaichiMain.changelog()`, L173.